src/legacy/ui/public/saved_objects/saved_object.d.ts

```diff
@@ -17,15 +17,13 @@
  * under the License.
```

Comment:
Github is wrong. This file was converted to ts, and saved_object.d.ts is new.  Crazy it can't figure this out.